app/routers/api/identity.py

```
@@ -0,0 +1,112 @@
+    """
+    keys = identity_service.export_keys_pem(db)
+    return {
+        "signing_public_key": keys["signing_public_key"],
+        "kx_public_key": keys["kx_public_key"],
+    }
+
+
+@router.post("/private-export", response_model=IdentityExportResponse)
+def export_private_keys(
+    request: PrivateExportRequest,
+    db: Session = Depends(get_db),
```

Ruff (0.14.14)

[warning] Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)